lib/ui/recommendation.js

```diff
@@ -6,101 +6,12 @@ const {classes: Cc, utils: Cu} = Components;
 
 Cu.import('resource://gre/modules/XPCOMUtils.jsm');
 XPCOMUtils.defineLazyModuleGetter(this, 'Services',
- 'resource://gre/modules/Services.jsm');
+  'resource://gre/modules/Services.jsm');
+XPCOMUtils.defineLazyModuleGetter(this, 'RecommendationRow',
+  'chrome://universalsearch-lib/content/ui/recommendation-row.js');
```

Comment:
Would you mind passing these in via the constructor, rather than invoking Cu.import and XPCOMUtils here?